src/app/lib/models/content_item.js

```diff
@@ -12,7 +12,7 @@
         initialize: function (attrs) {
             var providers = Object.assign(this.getProviders(), attrs.providers);
             this.set('providers', providers);
-            this.set('idAttribute', this.idAttribute);
+            this.set('idAttribute', attrs.uniqueId || this.get('uniqueId') || this.idAttribute);
```

Comment:
so we were getting into that code by many different paths... and Backbone init does weird things on some of those paths...